add stronger wording regarding respect in CoC (#142)
ubclaunchpad/docs: handbook/manifesto.md

All members must agree to and abide by the following policies:

> * Treat everyone in and outside of Launch Pad with respect - we do not condone any discrimination or intolerance against each other or any group of people.

Reviewer: ```suggestion
* Treat everyone with respect - we do not condone any form of discrimination or intolerance.
```

Author: I like this!
